website/client/components/appFooter.vue

```diff
@@ -99,125 +142,103 @@
 </style>
 
 <script>
+import axios from 'axios';
+import moment from 'moment';
+import { mapState } from 'client/libs/store';
 import gryphon from 'assets/svg/gryphon.svg';
 
+import modifyInventory from './modifyInventory';
+
 // const IS_PRODUCTION = process.env.NODE_ENV === 'production'; // eslint-disable-line no-process-env
```

Comment:
we should import this with `import()` when calling modifyInventory() so it's not loaded for all users